{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--350b3add-1ebc-4734-8178-566792051860",
    "created": "2020-05-26T20:30:01.255179Z",
    "modified": "2020-05-26T20:30:01.255179Z",
    "name": "CVE-2019-15032",
    "description": "Pydio 6.0.8 mishandles error reporting when a directory allows unauthenticated uploads, and the remote-upload option is used with the http://localhost:22 URL. The attacker can obtain sensitive information such as the name of the user who created that directory and other internal server information."
}